opacus/grad_sample/conv.py

```diff
@@ -82,6 +83,49 @@ def compute_conv_grad_sample(
 
     ret = {layer.weight: grad_sample.view(shape)}
     if layer.bias is not None:
-        ret[layer.bias] = torch.sum(backprops, dim=2)
+        ret[layer.bias] = torch.sum(backprops, dim=[-1, -2])
```

Comment:
What prompted this change?

Reply:
Usual batches of images are in the form BCHW, but for e.g. data augmentations you can have BACHW where 'A' is the number of augmentations.